Makefile

```diff
@@ -71,7 +71,7 @@ else
 export GOBIN ?= ./out/bin
 endif
 
-MAKE = $(RUN) make -e -f Makefile.core.mk
+MAKE = $(RUN) make --no-print-directory -e -f Makefile.core.mk
```

Comment:
This looks a little redundant: https://github.com/istio/common-files/commit/4328839147bab7f10dd5790bd95c275482de83d2

Reply:
removed